props/saber_sabersense_buttons.h

```diff
@@ -476,6 +501,45 @@ class SabersenseButtons : public PROP_INHERIT_PREFIX PropBase {
 #endif
   }
 
+  // PRESET SKIPPING
+  // Skip Value definable.
+  int GetNumberOfPresets() {
+    CurrentPreset tmp;
+    tmp.SetPreset(-1);
+    return tmp.preset_num + 1;
+  }
+
+  // Two Button system: Up/Down only - loose angle settings for ease of use.
+  bool MultiFontSkip(int skip_value) {
+      float angle = fusor.angle1();
+      int delta = 0;
+
+#if NUM_BUTTONS == 2
+      delta = (angle < -M_PI / 4) ? -skip_value : skip_value;
```

Comment:
shouldn't this be angle < 0 ? (above/below horizon)

Right now the cutoff is 45 degrees I think?